### PR TITLE
717 - Add metadata only download readme

### PR DESCRIPTION
**Pull request by avrohomgottlieb**
## Issue Number

https://github.com/AlexsLemonade/scpca-portal/issues/717

## Purpose/Implementation Notes

This PR adds a read me for metadata only downloads.

## Types of changes

What types of changes does your code introduce?

- New feature (non-breaking change which adds functionality)

## Functional tests

N/A

## Checklist

N/A

## Screenshots

N/A

### CODE REVIEW

**Comment by avrohomgottlieb** [comment]
PR was accidentally closed and branch was accidentally disposed of. GitHub is a mystery. We're back in business now, though 👍

File: api/scpca_portal/templates/readme/metadata_only.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+To cite data from {{ project_accession }}, please see the project abstract and publication information at [{{ project_accession }} page.]({{ project_url }})
+
+{% endfor %}
+
+### Citing the ScPCA Portal
+
+When citing the ScPCA Portal, please cite the following preprint:
+
+Hawkins A. G., J. A. Shapiro, S. J. Spielman, D. S. Mejia, D. V. Prasad, et al., 2024 The Single-cell Pediatric Cancer Atlas: Data portal and open-source tools for single-cell transcriptomics of pediatric tumors. _bioRxiv._ https://doi.org/10.1101/2024.04.19.590243
```
Comment (sjspielman) [inline api/scpca_portal/templates/readme/metadata_only.md:47]: Just noting inconsistent initial/last name order here

Comment (allyhawkins) [reply api/scpca_portal/templates/readme/metadata_only.md:47]: This is correct. We use [genetics citation style](https://paperpile.com/s/genetics-citation-style/).

Comment (sjspielman) [reply api/scpca_portal/templates/readme/metadata_only.md:47]: wow, I never knew that! cool

File: api/scpca_portal/templates/readme/metadata_only.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).
+
+## Contents
+
+{% if project_accession %}
+This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
+{% else %}
+This download includes associated metadata for samples from all projects currently available at time of download in the ScPCA portal.
+{% endif %}
+
```
Comment (allyhawkins) [inline api/scpca_portal/templates/readme/metadata_only.md:15]:
```suggestion
The metadata included in this download contains sample, library, and project-related metadata (e.g., age, sex, diagnosis, sequencing unit, etc.) along with any relevant processing metadata (e.g., software versions, filtering methods used, etc.). 
```
Probably useful to expand on the contents of what they can expect in the metadata slightly although the full description is in the docs link.

**Review by allyhawkins — COMMENTED** [review]
Just one minor comment but otherwise looks good!

File: api/scpca_portal/templates/readme/metadata_only.md (hunk 2 above)
Comment (avrohomgottlieb) [reply api/scpca_portal/templates/readme/metadata_only.md:15]: Great! Added.

File: api/scpca_portal/templates/readme/metadata_only.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Generated on: {{ date }}
+
+# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas
+
+The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).
+
+## Contents
+
+{% if project_accession %}
```
Comment (davidsmejia) [inline api/scpca_portal/templates/readme/metadata_only.md:10]: we shouldn't pass `project_accession` here we should expect:

`included_projects` - a list of tuples where the first item is the accession and the second is the url

We should instead check that the `included_projects` is greater than 1 and display the "else" content here
If it is only 1 just iterate over the tuples.

File: api/scpca_portal/templates/readme/metadata_only.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
+{% else %}
+This download includes associated metadata for samples from all projects currently available at time of download in the ScPCA portal.
+{% endif %}
```
Comment (davidsmejia) [inline api/scpca_portal/templates/readme/metadata_only.md:14]:
```suggestion
{% if entire_portal %}
This download includes associated metadata for samples from all projects currently available at time of download in the ScPCA portal.
{% else %}
{% for project_accession, project_url in included_projects %}
This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
{% endfor %}
{% endif %}
```

We can add a new attribute that says it is for everything in the portal.

File: api/scpca_portal/templates/readme/metadata_only.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- The data submitter using language provided as part of the project abstract (as applicable), the publication listed for the project (as applicable), or both.
+- The ScPCA Portal using the language below.
+
+For more information, please see [the How to Cite section of our documentation](https://scpca.readthedocs.io/en/stable/citation.html).
+
+### Citing this project
+
+{% for project_accession, project_url in included_projects %}
+To cite data from {{ project_accession }}, please see the project abstract and publication information at [{{ project_accession }} page.]({{ project_url }})
+{% endfor %}
```
Comment (avrohomgottlieb) [inline api/scpca_portal/templates/readme/metadata_only.md:41]: @davidsmejia
If there are multiple projects, do we want multiple line 40s, or do we want to handle this in a similar way to line 13?

Comment (davidsmejia) [reply api/scpca_portal/templates/readme/metadata_only.md:41]: nope this looks correct

File: api/scpca_portal/templates/readme/metadata_only.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas
+
+The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).
+
+## Contents
+
+{% if entire_portal %}
+This download includes associated metadata for samples from all projects currently available at time of download in the ScPCA portal.
+{% else %}
+This download includes associated metadata for samples from project {{ included_projects|markdown_links }} in the ScPCA portal.
```
Comment (davidsmejia) [inline api/scpca_portal/templates/readme/metadata_only.md:13]:
```suggestion
{% for proejct_accession, project_url in included_projects %}
This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
{% endfor %}
```

Let's just keep this simple. For now we only anticipate two use cases:
- All projects
- One project

We can rewrite it to make sense for an arbitrary amount of projects later.

File: api/scpca_portal/templates/readme/metadata_only.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
+{% load custom_filters %}
+Generated on: {{ date }}
+
+# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas
+
+The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).
+
+## Contents
+
+{% if entire_portal %}
```
Comment (davidsmejia) [inline api/scpca_portal/templates/readme/metadata_only.md:10]:
```suggestion
{% if len(included_projects) > 1 %}
```

**Review by davidsmejia — COMMENTED** [review]
I think we should keep the loops simple for now. We can open an issue later when we are actually approaching arbitrary projects in a download.

File: api/scpca_portal/templates/readme/metadata_only.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
+{% load custom_filters %}
```
Comment (davidsmejia) [inline api/scpca_portal/templates/readme/metadata_only.md:1]:
```suggestion
```

**Review by davidsmejia — APPROVED** [review]
LGTM after removing the custom filter loading

**Pull request merged**